script.js

```diff
@@ -47,3 +47,50 @@ function topFunction() {
 	document.body.scrollTop = 0;
 	document.documentElement.scrollTop = 0;
 }
+
+// TEAM MODALS
+
+// Overlay
+var overlay = document.getElementById("overlay");
+
+// close modal
+const closeModal = function () {
+	const modals = document.querySelectorAll(".modal");
+	modals.forEach(modal => {
+		modal.classList.add("hidden");
+	});
+	overlay.classList.add("hidden");
+	document.body.classList.remove("fixed");
+	topBtn.classList.remove("hidden");
+};
+
+var closeBtn = document.querySelectorAll(".close-btn");
+closeBtn.forEach(btn => {
+	btn.addEventListener("click", closeModal);
+});
+
+overlay.addEventListener("click", closeModal);
+
+// Open modal
+var teamMembers = document.querySelectorAll(".person");
+teamMembers.forEach(member => {
+	member.addEventListener("click", function () {
+		var modalId = member.getAttribute("data-modal");
```

Comment:
This is the magic that allows us to identify each list item uniquely:
`<li class="person" data-modal="lianna-modal">`